app/src/main/java/de/westnordost/streetcomplete/quests/bicycle_repair_station/BicycleRepairStationServiceItem.kt

```diff
@@ -0,0 +1,22 @@
+package de.westnordost.streetcomplete.quests.bicycle_repair_station
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.quests.bicycle_repair_station.BicycleRepairStationService.*
+import de.westnordost.streetcomplete.view.image_select.Item
+
+fun BicycleRepairStationService.asItem(): Item<BicycleRepairStationService> =
+    Item(this, iconResId, titleResId)
+
+val BicycleRepairStationService.titleResId: Int get() = when (this) {
+    PUMP -> R.string.quest_bicycle_repair_station_pump
+    TOOLS -> R.string.quest_bicycle_repair_station_tools
+    STAND -> R.string.quest_bicycle_repair_station_stand
+    CHAIN_TOOL -> R.string.quest_bicycle_repair_station_chain_tool
+}
+
+val BicycleRepairStationService.iconResId: Int get() = when (this) {
+    PUMP -> R.drawable.bicycle_repair_station_pump
```

Comment:
Those pictures looked tiny to me, but upon closed inspection that was just because they are too crowded by that text over them.  

Could we have text below the icon instead of over it, like in recycling quests? That would make it much more usable (as well as more consistent) IMHO:

![small_Screenshot_20241115_220206_StreetComplete Dev](https://github.com/user-attachments/assets/a5a8f106-b17d-4973-b813-e86c79f63416) ![small_Screenshot_20241115_215715_StreetComplete Dev](https://github.com/user-attachments/assets/99e70baa-74fe-43cf-bb69-f07803eac04c)

Or, if the text is going to be drawn over pictures, then make them bigger - i.e. put it in 2x2 instead 1x4 grid  (i.e. closer in looks to how sidewalk overlay icons look in 2+1, or like `surface` quest which is Nx3)?

Reply:
> as well as more consistent

AFAIK all photo selections work in this way

Reply:
> quest definitely does not look like that; the text is below the pictures there, not over them.

that is not **photo** selection

Reply:
> that is not **photo** selection, these drawings are not photos

oh, ok, thanks for clarification, I did not notice that subtle distinction. My mind sees to conflate image/photo as any graphical representation  :sweat_smile: 

Anyway, I see it's changed to `itemsPerRow = 3` now, so hopefully that helps enough.